src/controller/CHIPDeviceController.cpp

```diff
@@ -1733,6 +1733,51 @@ void DeviceCommissioner::OnDone(app::ReadClient *)
             return CHIP_NO_ERROR;
         }
     });
+
+    // Try to parse as much as we can here before returning, even if this is an error.
+    return_err = (err == CHIP_NO_ERROR) ? return_err : err;
+
+    err = mAttributeCache->ForEachAttribute(
+        app::Clusters::OperationalCredentials::Id, [this, &info](const app::ConcreteAttributePath & path) {
+            if (path.mAttributeId != app::Clusters::OperationalCredentials::Attributes::Fabrics::Id)
```

Comment:
This whole file is `using namespace chip::app::Clusters;`, no?
```suggestion
            if (path.mAttributeId != OperationalCredentials::Attributes::Fabrics::Id)
```
I would think.

Reply:
fixed